packages/dao/scripts/deploy.js

```diff
@@ -44,29 +45,75 @@ const getDeployer = async () => {
 module.exports = async (callback) => {
   try {
     const network = await getNetworkName();
+    const deployer = await getDeployer();
+
+    let api3Token, api3Pool;
+    let api3TokenAddress, timelockManagerAddress;
     if (network === "mainnet") {
-      // TODO: We should use specific daoId and do NOT create new token and pool instance
-      throw new Error("This script is not yet ready for mainnet deployment!");
+      // TODO: replace below with 0x0b38210ea11411557c13457D4dA7dC6ea731B88a
+      api3TokenAddress = "0x0000000000000000000000000000000000000001";
+      timelockManagerAddress = "0xFaef86994a37F1c8b2A5c73648F07dd4eFF02baA";
+      api3Pool = await Api3Pool.new(api3TokenAddress, timelockManagerAddress, {
+        gasPrice: gasPrice,
+      });
+      // Pass an API3DAOv1 proposal to call the token contract to authorize the pool contract as a minter
```

Comment:
https://client.aragon.org/#/api3daov1/
No, there's no compatibility, and this will be a fairly manual process. We'll be taking care of it, don't worry about it.